man/pro_object.Rd

```diff
@@ -17,10 +17,8 @@ A proData object
   \item{e_meta}{NULL}
```

Comment:
The object now has e_meta.  Not sure if an issue but I don't see an .Rd file for `pro_emeta` either

Reply:
Ah, yes, good catch. I added pro_emeta and updated the documentation. It had been missing because I got to the pro_object via protein rollup of the pep_object...so didn't have an explicit/separate pro_emeta file. Thanks!